kiliautoml/utils/pytorchvision/trainer.py

```diff
@@ -34,14 +37,23 @@ def train_model_pytorch(
     optimizer = optim.SGD(model.parameters(), lr=0.001, momentum=0.9)
 
     model = model.to(device)
-    dataset_sizes = {x: len(dataloaders[x].dataset) for x in ["train", "val"]}
-
     # Decay LR by a factor of 0.1 every 7 epochs
     scheduler = lr_scheduler.StepLR(optimizer, step_size=7, gamma=0.1)
 
     best_model_wts = copy.deepcopy(model.state_dict())
-    best_acc = 0.0
-    best_loss = float("inf")
+    best_val_acc = 0.0
```

Comment:
Sorry I missed that: that method is now way to big. I suggest a Metric object that would be initialized, updated, then rendered when the method returns.

Reply:
I can put one dictionary for train metrics and one for val metrics?